gammapy/maps/tests/test_wcsnd.py

```diff
@@ -378,3 +379,15 @@ def test_coadd_unit():
 
     assert_allclose(m1.data, 1.0001)
 
+def test_make_region_mask():
+    from regions import CircleSkyRegion
+    geom = WcsGeom.create(npix=(11,11), binsz=2,
```

Comment:
Suggest to do fewer pixels, so that printouts are nice if an error occurs.
This is what we had before:
https://github.com/gammapy/gammapy/blob/c209481f42edf675dc5d675cb2fd4ffd1ec9873d/gammapy/image/tests/test_core.py#L384

Another key question is if we return `dtype` of `bool` or `int`. Please add an assert to establish that.
I think `bool` makes sense, but then `write` fails because FITS can't handle it? If so, we could consider changing `write` to convert to `unit8` for `bool` maps.